soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1024

```diff
@@ -0,0 +1,26 @@
+# i.MX RT1024
+
+# Copyright (c) 2020, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_MIMXRT1024
+
+config SOC
+	default "mimxrt1024"
+
+config NUM_IRQS
+	default 142
+
+config ARM_DIV
+	default 0
+
+config AHB_DIV
+	default 0
```

Comment:
I am not sure, how it works for NXP, but I've never seen divider equal 0. @MaureenHelm  Is this okay?